clang/test/SemaCXX/invalid-requirement-requires-parameter.cpp

```diff
@@ -0,0 +1,15 @@
+// RUN: %clang -fsyntax-only -std=c++2a -Xclang -verify %s
```

Comment:
Can you move the test to `clang/test/CXX/expr/expr.prim/expr.prim.req/requires-expr.cpp`? I don't think this issue merits a new one